data/json/recipes/recipe_food.json

```diff
@@ -2343,18 +2277,8 @@
   "time": 10000,
   "autolearn": true,
   "batch_time_factors":[80, 4],
-  "qualities":[
-    {"id":"BOIL","level":1}
-  ], "tools": [
-    [
-      [ "hotplate", 2 ],
-      [ "char_smoker", 2 ],
-      [ "toolset", 2 ],
-      [ "atomic_coffeepot", -1 ],
-      [ "coffeemaker", 2],
-      [ "fire", -1 ]
-    ]
-  ],
+  "qualities":[ {"id":"BOIL","level":1} ],
+  "tools": [ [ [ "water_boiling_heat", 3, "LIST" ] ] ],
```

Comment:
Looks like you got the wrong line.

I think I replaced one 2 with 3 for consistency, but it was somewhere else.